lib/block-supports/typography.php

```diff
@@ -219,6 +219,178 @@ function gutenberg_typography_get_css_variable_inline_style( $attributes, $featu
 	return sprintf( '%s:var(--wp--preset--%s--%s);', $css_property, $css_property, $slug );
 }
 
+/**
+ * Internal method that checks a string for a unit and value and returns an array consisting of `'value'` and `'unit'`, e.g., [ '42', 'rem' ].
+ *
+ * @access private
+ *
+ * @param string $raw_value            Raw size value from theme.json.
+ * @param string $coerce_to            Coerce the value to rem or px. Default `'rem'`.
+ * @param number $root_font_size_value Value of root font size for rem|em <-> px conversion.
+ * @param array  $acceptable_units     An array of font size units.
+ * @return array                       An array consisting of `'value'` and `'unit'`, e.g., [ '42', 'rem' ]
+ */
+function gutenberg_get_typography_value_and_unit( $raw_value, $coerce_to = '', $root_font_size_value = 16, $acceptable_units = array( 'rem', 'px', 'em' ) ) {
```

Comment:
@ramonjd, what do you think about using an array of arguments here?

We don't have an official standard for the number of arguments, but I like the "2 or fewer" [suggestion from the Clean Code](https://github.com/jupeter/clean-code-php#function-arguments-2-or-fewer-ideally).

**Pros**
It's easier to add or deprecate arguments.

**Cons**
Requires a bit of boilerplate to handle defaults.

P.S. Check out [`wp_parse_args`](https://developer.wordpress.org/reference/functions/wp_parse_args/) and its usage in Core for [examples](https://developer.wordpress.org/reference/functions/register_theme_feature/).

Reply:
Sounds like a good optimization, thanks @Mamaduka !